tests/application/assets/Container.ts

```diff
@@ -48,12 +46,95 @@ export const containerAssetDefinition: AssetModelDefinition = {
   defaultMetadata: {
     height: 20,
   },
+  metadataDetails: {
+    extTemp: {
+      group: "environment",
+      locales: {
+        en: {
+          groupFriendlyName: "Environment",
+          friendlyName: "External Temperature",
+          description: "The temperature outside the container"
+        },
+        fr: {
+          friendlyName: "Température Externe",
```

Comment:
No groupFirnedlyName for the French translation ?

Reply:
same as said earlier, it's an error, changed my mind when developping and not change every where sry